examples/yoti_example_django/yoti_example/urls.py

```diff
@@ -16,10 +16,11 @@
 from django.conf.urls import url
 from django.contrib import admin
 
-from .views import IndexView, AuthView
+from .views import IndexView, AuthView, DynamicShareView
 
 urlpatterns = [
-    url(r'^$', IndexView.as_view(), name='index'),
-    url(r'^yoti/auth/$', AuthView.as_view(), name='auth'),
-    url(r'^admin/', admin.site.urls),
+    url(r"^$", IndexView.as_view(), name="index"),
+    url(r"^yoti/auth/$", AuthView.as_view(), name="auth"),
+    url(r"^admin/", admin.site.urls),
+    url(r"^dynamic_share/$", DynamicShareView.as_view(), name="dynamic_share"),
```

Comment:
I think as @echarrod mentioned on another PR, the URL pattern should use a hyphen rather than an underscore

```suggestion
    url(r"^dynamic-share/$", DynamicShareView.as_view(), name="dynamic_share"),
```